lib/presentation/speed_dial/my_speed_dial.dart

```diff
@@ -51,18 +59,33 @@ class MySpeedDial extends StatelessWidget {
                 animatedIcon: AnimatedIcons.menu_home,
                 animatedIconTheme: IconThemeData(size: 28),
                 children: [
-                  // TODO: 今後実装する
-                  // SpeedDialChild(
-                  //   onTap: () {
-                  //     //TODO: シェアする処理を書く
-                  //   },
-                  //   child: Icon(Icons.share),
-                  //   backgroundColor: Colors.red,
-                  //   label: 'シェアする',
-                  //   labelBackgroundColor: Colors.red,
-                  //   labelStyle:
-                  //       TextStyle(color: Colors.white, fontWeight: FontWeight.bold),
-                  // ),
+                  //share機能
+                   SpeedDialChild(
+                     onTap: () async {
+                       //呼び出し元が、個人ページ or 全体ページかでキャプチャするwidgetのkeyを変更
+                       shareFlag == ShareFlag.MapPage
+                           ? globalKey = mapPageConvertWidgetToImageKey
+                           : globalKey = individualMapPageConvertWidgetToImageKey;
+                       //shareしたい画像のキャプチャ処理
+                       await model.doCapture(globalKey);
+                       //キャプチャした画像のバイトデータを格納
+                       final byteData = model.byteData;
+                       //share機能
+                       await Share.file(
+                         "share image",
+                         "Location47.jpeg",
+                         byteData.buffer.asUint8List(),
+                         "image/jpeg",
+                         text: "〜 Location47 〜\n47都道府県の達成状況\nhttps://bit.ly/3rKX0sn",
+                       );
+                     },
+                     child: Icon(Icons.share, color: Colors.white,),
+                     backgroundColor: Colors.red,
+                     label: 'シェアする',
+                     labelBackgroundColor: Colors.red,
```

Comment:
```suggestion
                    backgroundColor: Color(0xFF00CA71),
                    label: 'シェアする',
                    labelBackgroundColor: Color(0xFF00CA71),
```

位置情報送信後のShareボタンと同じ色が良いです！